### PR TITLE
Refactoring ideas -  Review

### PR DESCRIPTION
**Pull request by Venyla**

### CODE REVIEW

File: chapters/refactoringIdeas.typ [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
 
 == Requirement Transformation <idea_requirement_transformation>
 A refactoring could be provided to transform a function template using constraints between alternate forms.
-@transformation_idea_listing shows the different forms.
+@transformation_idea_listing shows different variations of a function template.
 They all result in an identical function signature.
 
 The benefit of this is in many cases a more readable function declaration.
-In our opinion the `requires` clause just causes noise, so the versions on the left (@transformation_idea_listing) should be preferred.
+For more readable code some developers prefer to remove unnecessary code parts like the `requires` keyword.
```
Comment (jeremystucki) [inline chapters/refactoringIdeas.typ:18]:
```suggestion
For more readable code some developers prefer to remove unnecessary code like the `requires` keyword.
```

File: chapters/refactoringIdeas.typ [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+The versions on the left in @transformation_idea_listing show how the code looks like without resulting in the same code logic.
```
Comment (jeremystucki) [inline chapters/refactoringIdeas.typ:19]: d'Logik blibt doch glich?

Comment (Venyla) [reply chapters/refactoringIdeas.typ:19]: "resulting in the same code logic" 
Chum ned ganz drus was d meinsch

Comment (jeremystucki) [reply chapters/refactoringIdeas.typ:19]: "without resulting" hesch gschrebe

Comment (jeremystucki) [reply chapters/refactoringIdeas.typ:19]: oh und "code logic" würi no zu "logic" ändere

Comment (Venyla) [reply chapters/refactoringIdeas.typ:19]:
```suggestion
The versions on the left in @transformation_idea_listing show how the code looks like without the keyword resulting in the same logic.
```

**Review by jeremystucki — APPROVED** [review]

**Pull request merged**